src/Parallel/distance.jl

```diff
@@ -1,20 +1,51 @@
 # used in shortest path calculations
 
 function eccentricity(
+    g::AbstractGraph,
+    vs=vertices(g),
+    distmx::AbstractMatrix{T}=weights(g);
+    parallel=:distributed,
+) where {T<:Number}
+    return if parallel === :threads
+        threaded_eccentricity(g, vs, distmx)
+    elseif parallel === :distributed
+        distr_eccentricity(g, vs, distmx)
+    else
+        error(
+            "Unsupported parallel argument '$(repr(parallel))' (supported: ':threads' or ':distributed')",
+        )
+    end
+end
+
+function distr_eccentricity(
     g::AbstractGraph, vs=vertices(g), distmx::AbstractMatrix{T}=weights(g)
 ) where {T<:Number}
     vlen = length(vs)
     eccs = SharedVector{T}(vlen)
     @sync @distributed for i in 1:vlen
-        eccs[i] = maximum(Graphs.dijkstra_shortest_paths(g, vs[i], distmx).dists)
+        d′ = Graphs.dijkstra_shortest_paths(g, vs[i], distmx)
```

Comment:
I am a bit unsure about the usage of `d′` - it is hardly distinguishable from `d'` - in that case `'` is the adjoint operator.